build.gradle

```diff
@@ -183,9 +183,11 @@ configurations {
     runtimeClasspath
 }
 
+// Spring Boot 3.2.4 pulls in opentelemetry-bom 1.31.0.
+// It must have version >= 1.34.1 for compatibility with terra-common-lib 1.1.10:
+ext['opentelemetry.version'] = '1.36.0'
```

Comment:
So this is defined and used by some mechanism outside this gradle build file? Normally when we I've seen an `ext[]` definition it's for use directly in the same file.

Reply:
Spring IO and Spring Boot BOMs use properties to define versions of their managed dependencies.  This is what I've gotten from their docs:

https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/#managing-dependencies.dependency-management-plugin.customizing
https://docs.spring.io/platform/docs/current/reference/html/getting-started-overriding-versions.html